app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors \
+    import NotVaccinatedError, NotWearingMaskError, OutdatedVaccineError
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError")
```

Comment:
the error text should be more informative: `raise NotVaccinatedError("The visitor is not vaccinated")`